doc/example/model_selection.ipynb

```diff
@@ -0,0 +1,828 @@
+    "from petab_select import Criterion, Method, ESTIMATE\n",
+    "petab_select_problem = petab_select.Problem.from_yaml('model_selection/petab_select_problem.yaml')\n",
+    "\n",
+    "\n",
+    "# Helpers for plotting\n",
+    "def get_labels(models):\n",
+    "    labels = {\n",
+    "        model.get_hash(): str(model.model_subspace_id)\n",
+    "        for model in models\n",
+    "    }\n",
+    "    return labels\n",
+    "\n",
+    "\n",
+    "def get_digraph_labels(models, criterion):\n",
+    "    zero = min(model.get_criterion(criterion) for model in models)\n",
+    "    labels = {\n",
+    "        model.get_hash(): f'{model.model_subspace_id}\\n{model.get_criterion(criterion) - zero:.2f}'\n",
+    "        for model in models\n",
+    "    }\n",
+    "    return labels"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "import logging\n",
+    "import pypesto.logging\n",
+    "pypesto.logging.log(level=logging.INFO, name='pypesto.select', console=True)\n",
+    "import petab\n",
+    "\n",
+    "pypesto_select_problem_1 = pypesto.select.Problem(petab_select_problem=petab_select_problem)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Models can be selected with a model selection algorithm (here: [forward](https://en.wikipedia.org/wiki/Stepwise_regression)) and a comparison criterion (here: [AIC](https://en.wikipedia.org/wiki/Akaike_information_criterion)). The forward method involves iterations where each iteration tests all models with one additional estimated parameter.\n",
+    "\n",
+    "As no initial model is specified here, a virtual initial model with no estimated parameters is automatically used to find the \"smallest\" (in terms of number of estimated parameters) models. In this example, this is the model `M1_0`, which has no estimated parameters."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "--------------------New Selection--------------------\n",
+      "--------------------New Selection--------------------\n",
+      "Predecessor model ID\tModel ID\tCriterion ID\tPredecessor model criterion\tModel criterion\tCriterion difference\tAccept\n",
+      "Predecessor model ID\tModel ID\tCriterion ID\tPredecessor model criterion\tModel criterion\tCriterion difference\tAccept\n",
+      "Using existing amici model in folder /home/dilan/Documents/future_annex/model_selection/packages/pyPESTO/doc/example/amici_models/caroModel_linear.\n",
+      "100%|██████████| 10/10 [00:01<00:00,  9.29it/s]\n",
+      "649bcd92d8da15883a6709f3233e277e173aa6ce7ff1eda3f32d342a0f3a0e63ddd6e3764eab0c730dfb96440bab731ee67d412295a48e1e9949b3cc10935f28\tvirtual_initial_model\tCriterion.AIC\t36.975\tNone\tNone\tTrue\n",
```

Comment:
what are these blobs? they seem not toooo informative ...

Reply:
Removed irrelevant logging output, and changed the model IDs to be more informative in this output.